content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -106,60 +93,71 @@ This example uses the default provisioner for Minikube. Please update the follow
 
 {{< codenew file="application/cassandra/cassandra-statefulset.yaml" >}}
 
-1. Update the StatefulSet if necessary.
-1. Create the Cassandra StatefulSet from the `cassandra-statefulset.yaml` file:
+Create the Cassandra StatefulSet from the `cassandra-statefulset.yaml` file:
 
-    ```shell
-    kubectl apply -f https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml
-    ```
+```shell
+# Use this if you are able to apply cassandra-statefulset.yaml unmodified
+kubectl apply -f https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml
+```
+
+If you need to modify `cassandra-statefulset.yaml` to suit your cluster, download
+https://k8s.io/examples/application/cassandra/cassandra-statefulset.yaml and then apply
+that manifest, from the folder you saved the modified version into:
+```shell
+# Use this if you needed to modify cassandra-statefulset.yaml locally
+kubectl apply -f cassandra-statefulset.yaml
+```
 
-## Validating The Cassandra StatefulSet
+
+## Validating the Cassandra StatefulSet
 
 1. Get the Cassandra StatefulSet:
 
     ```shell
     kubectl get statefulset cassandra
     ```
 
-    The response should be:
+    The response should be similar to:
 
     ```
     NAME        DESIRED   CURRENT   AGE
     cassandra   3         0         13s
     ```
 
-    The `StatefulSet` resource deploys Pods sequentially.  
+    The `StatefulSet` resource deploys Pods sequentially.
 
 1. Get the Pods to see the ordered creation status:
 
     ```shell
     kubectl get pods -l="app=cassandra"
     ```
-       
-    The response should be:
-       
+
+    The response should be similar to:
+
     ```shell
     NAME          READY     STATUS              RESTARTS   AGE
     cassandra-0   1/1       Running             0          1m
     cassandra-1   0/1       ContainerCreating   0          8s
     ```
-    
-    It can take several minutes for all three Pods to deploy. Once they are deployed, the same command returns:
-    
+
+    It can take several minutes for all three Pods to deploy. Once they are deployed, the same command
+    returns output similar to:
+
     ```
     NAME          READY     STATUS    RESTARTS   AGE
     cassandra-0   1/1       Running   0          10m
     cassandra-1   1/1       Running   0          9m
     cassandra-2   1/1       Running   0          8m
     ```
 
-3. Run the Cassandra [nodetool](https://wiki.apache.org/cassandra/NodeTool) to display the status of the ring.
+3. Run the Cassandra [nodetool](https://wiki.apache.org/cassandra/NodeTool) inside the first Pod, to
+   display the status of the ring.
```

Comment:
@sftim , I think the link has changed,
https://cwiki.apache.org/confluence/display/CASSANDRA2/NodeTool